internal/controllers/topology/cluster/cluster_controller.go

```diff
@@ -198,12 +198,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Re
 
 // reconcile handles cluster reconciliation.
 func (r *Reconciler) reconcile(ctx context.Context, s *scope.Scope) (ctrl.Result, error) {
+	log := ctrl.LoggerFrom(ctx)
 	var err error
 
 	// Get ClusterClass.
 	clusterClass := &clusterv1.ClusterClass{}
 	key := client.ObjectKey{Name: s.Current.Cluster.Spec.Topology.Class, Namespace: s.Current.Cluster.Namespace}
 	if err := r.Client.Get(ctx, key, clusterClass); err != nil {
+		if apierrors.IsNotFound(err) {
+			log.V(6).Info(fmt.Sprintf("could not find clusterClass %s/%s for this object", s.Current.Cluster.Namespace, s.Current.Cluster.Spec.Topology.Class))
```

Comment:
```suggestion
			log.V(6).Info("could not find ClusterClass", "ClusterClass", klog.KRef(s.Current.Cluster.Namespace, s.Current.Cluster.Spec.Topology.Class))
```
nit: should be better for k/v logging